### PR TITLE
unlabel week/month related tests

### PR DESCRIPTION
**Pull request by ziransun**
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our README.md: https://github.com/web-platform-tests/wpt-metadata/blob/master/README.md 
2. Please label this pull request `do not merge yet` upon creation because the auto-merge feature is enabled in this repository. If this pull request is finished, feel free to assign foolip/kyleju as reviewers, or we will review and merge them automatically.

### CODE REVIEW

**Comment by ziransun** [comment]
This is to remove the interop-2022-forms label for some month/week related tests. These tests pass in Chromium and Firefox but fail in Safari due to the lack of month/week support. In WebKit month/week support for forms is not in the scope of interop-2022.

File: html/semantics/forms/constraints/META.yml [hunk 1]
```diff
@@ -27,15 +26,13 @@ links:
...
-        - test: form-validation-reportValidity.html
         - test: form-validation-validity-badInput.html
         - test: form-validation-validity-rangeOverflow.html
         - test: form-validation-validity-tooShort.html
         - test: form-validation-validity-valueMissing.html
         - test: input-pattern-dynamic-value.html
         - test: radio-valueMissing.html
-        - test: form-validation-validity-customError.html
-        - test: form-validation-validity-stepMismatch.html
+        - test: form-validation-validity-customError.html
```
Comment (foolip) [inline html/semantics/forms/constraints/META.yml:35]: Trailing whitespace makes it look like a change in the diff:

```suggestion
        - test: form-validation-validity-customError.html
```

File: html/semantics/forms/the-input-element/META.yml [hunk 2]
```diff
@@ -222,8 +220,7 @@ links:
     - test: input-validationmessage.html
     - test: maxlength-number.html
     - test: input-type-checkbox.html
-    - test: input-valueasnumber-invalidstateerr.html
-    - test: input-valueasnumber-stepping.html
+    - test: input-valueasnumber-invalidstateerr.html
```
Comment (foolip) [inline html/semantics/forms/the-input-element/META.yml:223]:
```suggestion
    - test: input-valueasnumber-invalidstateerr.html
```

File: html/semantics/forms/the-input-element/META.yml [hunk 3]
```diff
@@ -232,8 +229,7 @@ links:
     - test: show-picker.html
     - test: valueMode.html
     - test: hidden.html
-    - test: hidden-charset-case-sensitive.html
-    - test: input-valueasdate.html
+    - test: hidden-charset-case-sensitive.html
```
Comment (foolip) [inline html/semantics/forms/the-input-element/META.yml:232]:
```suggestion
    - test: hidden-charset-case-sensitive.html
```

**Review by foolip — APPROVED** [review]
Thanks @ziransun! I've commented in https://github.com/web-platform-tests/interop-2022/issues/11 so that everyone is aware.

**Pull request merged**